packages/phoenix-event-display/src/managers/state-manager.ts

```diff
@@ -80,8 +82,8 @@ export class StateManager {
       phoenixMenu: this.phoenixMenuRoot.getNodeState(),
       eventDisplay: {
         cameraPosition: this.activeCamera.position.toArray(),
-        clippingAngle: this.clippingEnabled.value
-          ? this.clippingAngle.value
+        startClippingAngle: this.clippingEnabled.value
```

Comment:
Sorry, I didn't notice it yesterday. Could you please also add the opening clipping angle to this state?

Reply:
Sure.